examples/demo.py

```diff
@@ -0,0 +1,25 @@
+# -------------------------------------------------------------------------
+#
+#  Copyright (c) 2005, Enthought, Inc.
+#  All rights reserved.
+#
+#  This software is provided without warranty under the terms of the BSD
+#  license included in enthought/LICENSE.txt and may be redistributed only
+#  under the conditions described in the aforementioned license.  The license
+#  is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+#  Thanks for using Enthought open source!
+#
+#  Author: David C. Morrill
+#  Date:   09/15/2005
+#
+# -------------------------------------------------------------------------
+
+""" Run the Traits UI demo.
+"""
+
+from __future__ import absolute_import
+
+from traitsui.extras.demo import demo
```

Comment:
It looks like this demo machinery in traitsui has the window title "Traits UI Demos" hardcoded. That's definitely what we want/expect but i'm not sure what the best way ahead is - Do we duplicate all of the demo-related functionality from traitsui in enable? Do we simply make the window title an argument to the `demo` function in traitsui (and then use a new release in enable)

Reply:
Make the window title customizable in the traitsui.extras.demo.